packages/constants/src/constants.ts

```diff
@@ -169,5 +169,6 @@ export interface ApplicationWindow extends Window {
     useFullRedirectsForLinks?: boolean;
     // Properties for OIDC-like workflow for development
     __DEVELOPMENT__?: ApplicationOidcForDevelopmentConfig;
+    enableFeatureConfigurationFetching?: boolean;
```

Comment:
...that's why I added it here.

Reply:
Can you move it below `useFullRedirectsForLinks`?
This way we keep the props "grouped". Thanks

Reply:
Sure. I moved it to `enableSignUp` cause those two look similar.